tests/client/services/demandeDeContact.service.test.ts

```diff
@@ -1,4 +1,4 @@
-import { aHttpClientService } from '@tests/fixtures/client/services/httpClientService.fixture';
+import { anHttpClientService } from '@tests/fixtures/client/services/httpClientService.fixture';
```

Comment:
""An" goes before words that begin with vowels
Exceptions
Use "an" before a slient or unsounded "h." Because the "h" does not have any phonetic representation or audible sound, the sound that follows the article is a vowel; consequently, "an" is used."

Ici h est suivi d'un t qui est une consonne, donc --> aHttpClientService()
Pourquoi on ne les appelerait pas httpClientServiceFixture() pour résoudre tous ces pbs (car ce n'est pas le seul endroit où il y a ce genre d'inexactitude)